csaf_2.1/prose/edit/src/schema-elements-01-defs-03-full-product-name.md

```diff
@@ -75,6 +75,13 @@ and `x_generic_uris`, one is mandatory.
       }
 ```
 
+A helper to identify the product SHALL identify the product described by the `name` in its entirety.
+A matching algorithm MUST be able to rely on the `product_identification_helper` information during the matching without having to
```

Comment:
"MUST be able to rely on" reads like a compliance statement an algorithm has to fulfill while I guess meant is motivating the SHALL (compliance statement) on the "helper", right? 

What about:
>A matching algorithm matches solely on the `product_identification_helper` information of the product and does not need to check e.g. the branches hierarchy leading to the product.

Reply:
> What about:
> 
> > A matching algorithm matches solely on the `product_identification_helper` information of the product and does not need to check e.g. the branches hierarchy leading to the product.

That wouldn't be 100% correct: The matching algorithm needs to take the categorized strings (aka  branches hierarchy leading to the product) into account if it doesn't have a PIH in the asset database to match on. But if it does, it must be able to match just on this.

What about:

```suggestion
A matching algorithm has to be able to rely on the `product_identification_helper` information during the matching without having to
``